src/RapidezStatamicServiceProvider.php

```diff
@@ -136,7 +130,12 @@ public function bootComposers() : self
                     ->where('collection', 'products')
                     ->where('site', $this->getSiteHandleByStoreId())
                     ->where('linked_product', config('frontend.product.sku'))
-                    ->first();
+                    ->first()
+
+                    // As the "product content" collection is removed
+                    // with it's blueprint we need to specify the
+                    // runway blueprint, but this doesn't work.
+                    ->blueprint('runway/product');
```

Comment:
We don't need the blueprint here as we're just querying the database for the entry, only important thing is to get the right entry and that's handled in the where clause above.

Reply:
When you're going to use the data on the frontend (with: https://github.com/rapidez/statamic#showing-content-on-categories-and-products) you'll end up with errors because the blueprint can't be found. When you put the old `product` blueprint here it's working fine.

Reply:
So it does also work with the `runway/product ` blueprint instead of the old `product` one?

Reply:
Nope, as mentioned in the comment. Maybe the path is incorrect, didn't dive into this further.